src/components/Pagination/pagination.vue

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+import { StreamingService } from '@/views/streaming.service';
+import router from '@/router';
+
+export default {
+    data() {
+        return {
+            streamings: new Array < StreamingService > (),
```

Comment:
Quando for tipar alguma variável, nao precisa de espaços entre os simbolos de < e >, ficaria  "Array\<StreamingService\>()"